### DISCUSSION TITLE
Encountering Attribute Error in PyBaMM v22.8: Seeking Solution for 'beta_LAM_dimensional2' Issue

### DISCUSSION THREAD

**Discussion by kimo707** [pybamm-team/PyBaMM#3924] [Q&A]
Hello, I'm having issues trying to display aging. However, I encounter the following error: 'DomainLithiumIonParameters' object has no attribute 'beta_LAM_dimensional2'. Currently, I am using PyBaMM version v22.8.. Could you please tell me how I should solve this problem? Thank you

**Comment by valentinsulzer** [comment]
Looks like a typo in your code

**Reply by kimo707** [reply]
I'm working on adapting my code for Accelerated Battery Lifetime Simulations to a newer software version. Unfortunately, the original code was developed with version 22.8, and I've encountered significant challenges due to changes in the variable names and structures by version 23.4.1. My main struggle involves updating parameters such as the Negative Electrode LAM (Loss of Active Material) constants (both the first and second proportional terms, expressed in [s-1]), and the Negative Electrode's critical stress. This issue also extends to the Positive Electrode parameters.

another issue, I utilized the ElectrodeSOH model from PyBaMM's lithium-ion battery module like this:
esoh_model = pybamm.lithium_ion.ElectrodeSOH()
esoh_sim = pybamm.Simulation(esoh_model, parameter_values=parameter_values)

However, this approach is no longer exist in 23.4.1 and in the current  version also . As a workaround, I started using pybamm.lithium_ion.get_min_max_stoichiometries(parameter_values, param) to derive the minimum and maximum stoichiometries (x_0, x_100, y_0, y_100 ), but I'm unsure how to adapt this function to update these variables effectively. I want to update the  input data  {"Q_n": Q_n, "Q_p": Q_p, "Q_Li": n_Li}, but I'm uncertain about the correct way to incorporate these variables into the get_min_max_stoichiometries function to update Q_n and Q_p before the estimation of (x_0, x_100, y_0, y_100 ) again. Could you provide guidance on how to adjust this function or perhaps there is other function for that ?
